awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -47,6 +47,13 @@ def is_s3_url(url):
         return False
 
 
+def is_https_s3_url(url):
+    if isinstance(url, six.string_types) \
+        and url.startswith("https://s3.amazonaws.com/"):
```

Comment:
Do we need to account for virtual host style addressing or does cfn not handle this either?  e.g `https://mybucket.s3.amazonaws.com/`?